apps/designer/app/designer/shared/image-manager/image-thumbnail.tsx

```diff
@@ -59,55 +41,41 @@ const ThumbnailContainer = styled(Box, {
 });
 
 const Thumbnail = styled(Box, {
-  width: "$spacing$19",
+  width: "100%",
   height: "$spacing$19",
```

Comment:
since you are not using your constant THUMBNAIL_WIDTH nothing stops these to mismatch, I would use it

Reply:
Here width is coming from parent `<Grid columns={3} gap={2}>` we must not use `width` in 99% of cases. This `width` caused a bug with `Animation` wasn't centered. I will add other comment for `const` or even remove it and add comment to `Image` prop

Reply:
Changed comment onto
https://github.com/webstudio-is/webstudio-designer/blob/7abe851a8c0e9d3ca2a9ec9aea14859c0043f6ae/apps/designer/app/designer/shared/image-manager/image-thumbnail.tsx#L97-L98